packages/playwright-core/src/server/registry/index.ts

```diff
@@ -748,7 +748,7 @@ export class Registry {
 
     const downloadFileName = `playwright-download-${descriptor.name}-${hostPlatform}-${descriptor.revision}.zip`;
     await downloadBrowserWithProgressBar(title, descriptor.dir, executablePath, downloadURLs, downloadFileName).catch(e => {
-      throw new Error(`Failed to download ${title}, caused by\n${e.stack}`);
+      throw new Error(`Failed to download ${title}, caused by\n${e.stack}\nIf this persists, please check here for a possible outage: https://azure.status.microsoft/en-us/status`);
```

Comment:
As of 1.22 we are going to try 3 different CDN networks, so Microsoft's one being down should not affect Playwright's ability to fetch the browsers. If you are still experiencing this issue, chances are you are on an old Playwright.

Reply:
Thanks for the update, I'm glad to hear that you've taken steps to improve resilience!

(We currently happen to be on 1.21.0 in the affected project. 😅)